README.md

```diff
@@ -29,6 +29,53 @@ GIT_SSH-KEY=-----BEGIN RSA PRIVATE KEY----- Hgiud8z89ijiojdobdikdosaa+hnjk789hds
 
 Please note there is a space ` ` between `-----BEGIN RSA PRIVATE KEY-----` and the key as well as between the key and `----END RSA PRIVATE KEY-----`. If you omit these spaces the container will not be able to read the private key.
 
+## Configuration using dhall
+
+Prerequisite:
+- Install dhall and dhall-to-yaml, can find details [here](https://docs.dhall-lang.org/tutorials/Getting-started_Generate-JSON-or-YAML.html#installation).
+
+The properties mentioned in the configuration section can be configured in the file dsl/Env/EnvConfig.dhall
```

Comment:
q: In run.sh the file that is read is ./AppConfig.dhall. Here you write EnvConfig.dhall. Which is correct?